### ISSUE TITLE
gui.vBox requires scrollbar

### ISSUE THREAD

**Issue by leaf918** [biolab/orange3#6460]
**What's your use case?**

I am creating a new addon in orange3, when a vbox is created  ,it needs a scrollbar ,because it has lot of items in it.
'''python
self.propertyBox = gui.vBox(self.controlArea, "Property")
'''

![image](https://github.com/biolab/orange3/assets/170311/b1d0625a-d964-4f24-8271-1a09d83d39ae)

**What's your proposed solution?**

**Are there any alternative solutions?**

**Comment by janezd** [comment]
Module gui contains a set of utility functions to reduce the amount of code needed to build interfaces with Qt. When they are insufficient or when using pure Qt is cleaner, one should use Qt directly.

Either use Qt directly or simplify your interface. In your case, I would put the labels in the same row as line edits, and I'd use QFormLayout. Also, I'd use labels with friendly names, that is "Enable last sample" not "enable-last-sample".

**Comment by leaf918** [comment]
Of course, labels "Enable last sample" is more friendly , it's not label only but the CLI parameter too, so I think keeping it to same name of gstreamer plugins' property is better.
